contracts/LidoMatic.sol

```diff
@@ -179,17 +179,22 @@ contract LidoMatic is AccessControlUpgradeable, ERC20Upgradeable {
 
             lastWithdrawnValidatorId++;
         } else {
+            // If delegation did not happen yet then withdraw Matic equal to StMatic balance
```

Comment:
do we need this, because the amount always should be in Matic
The function => getUserBalanceInMATIC() returns to us the right amount in Matic even if we delegated or not yet.